browser/webpack.mocha.js

```diff
@@ -16,6 +17,14 @@ WebpackConfig.module.rules[2].use.options.presets[0][1].exclude = [
     '@babel/plugin-transform-regenerator'
 ];
 
-delete WebpackConfig.node.util;
+// Used in testing.
+WebpackConfig.plugins.push(new Webpack.ProvidePlugin({
+  process: 'process/browser',
+}));
```

Comment:
Looks like we can remove `process` by doing this.
```suggestion
WebpackConfig.plugins.push(new Webpack.DefinePlugin({
  'process.env.NODE_DEBUG': false,
}));
```

Reply:
updated :)